editor.go

```diff
@@ -0,0 +1,168 @@
+package survey
+
+import (
+	"bytes"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"runtime"
+
+	"github.com/AlecAivazis/survey/core"
```

Comment:
All of the imports to `survey` need to come from `gopkg.in/AlecAivazis/survey.v1` so that there is only one copy being used (allowing for editing the icons and what not)

Reply:
OK, though I don't think it's a good idea, as it breaks the meaning of version control. It breaks anyone who wants to `go get` the package at master branch. (It seems like a known bug in gopkg.in)

Reply:
Hm so i'm still getting used to the proper patterns when pushing packages that rely on `gopkg.in` I will do some research to figure out what the right thing to do is. However, i think hosting the package through `gopkg.in` and being able to control which version is being pulled seems like it's worth this slight complexity. 

Let's make the imports consistent now and if we decide they need to go back to the normal `github.com/...` then we can change them all together